iml-system-test-utils/src/lib.rs

```diff
@@ -516,8 +519,7 @@ pub async fn deploy_servers(config: Config) -> Result<Config, TestError> {
     Ok(config)
 }
 
-pub async fn configure_docker_network(config: &Config) -> Result<(), TestError> {
-    let host_list = config.profile_map.to_server_list();
```

Comment:
Why change this?

Reply:
I'm curious as to what the value is here. It doesn't look like it's causing an issue though.

Reply:
It's equivalent to what it was before.